cpp/src/gandiva/precompiled/types.h

```diff
@@ -132,6 +132,10 @@ gdv_int64 div_int64_int64(gdv_int64 context, gdv_int64 in1, gdv_int64 in2);
 gdv_float32 div_float32_float32(gdv_int64 context, gdv_float32 in1, gdv_float32 in2);
 gdv_float64 div_float64_float64(gdv_int64 context, gdv_float64 in1, gdv_float64 in2);
 
+gdv_float32 round_float32_int32(gdv_float32 number, gdv_int32 precision);
```

Comment:
looks like precision is misnamed..can you please rename this too

Reply:
Oh right sure. Missed that.